src/pykiso/lib/robot_framework/uds_auxiliary.py

```diff
@@ -162,6 +165,28 @@ def write_data(
         aux = self._get_aux(aux_alias)
         return aux.write_data(parameter, value)
 
+    @keyword(name="Start tester present sender")
+    def start_tester_present_sender(self, period: int, aux_alias) -> None:
```

Comment:
actually the code of start_tester_present_sender and stop_tester..  could be  in auxiliary itself...
So a normal itf user could use the feature without context manager...
Here you would just call the method from the auxiliary

Reply:
Actually that was just a trick to have a behaviour a bit similar to the normal uds aux as robot framework syntax does not allow the use of a context manager. Not sure there is a use case were it is better not to use a context manager